CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.12.1] - 2018-08-28
+
+### Fixed
+
+* Bug fix: Stop attempts to store invalid events (i.e. errors etc), since this raises an exception.
```

Comment:
I find this message a bit confusing... It gives the impression that we don't store invalid events (error, INVALID quality factor) any longer... This is not true, the bug fix actually restores the possibility to store the first non-consecutive error events and to store again the attribute quality factor when it is set to INVALID.